docs/publisher.md

```diff
@@ -0,0 +1,23 @@
+Publisher
+=========
+The Publisher block sends signals to the configured topic. Publisher blocks can send signals on a configured topic to a subscriber block in any instance within the same system. Signals must be valid JSON.
```

Comment:
I think the  old description includes important info about pickling, any reason it was omitted?